api-catalog-ui/frontend/src/components/Wizard/WizardComponents/WizardInputs.test.jsx

```diff
@@ -320,94 +334,103 @@ describe('>>> WizardInputs tests', () => {
                 },
-            }],
+            ],
         };
-        const wrapper = enzyme.shallow(
-            <WizardInputs data={dummyData} />
-        );
+        const wrapper = enzyme.shallow(<WizardInputs data={dummyData} />);
         const instance = wrapper.instance();
         const result = instance.renderInputs(dummyData.content[0]);
         expect(result[0]).toEqual(null);
     });
     it('should render if dependency is satisfied', () => {
         const dummyData = {
             text: 'Category',
-            content: [{
-                test: { value: '', question: 'Why?', dependencies: { scheme: 'schemeValue' } },
-                scheme: { value: 'schemeValue', question: 'Who?' },
-            }],
+            content: [
+                {
+                    test: { value: '', question: 'Why?', dependencies: { scheme: 'schemeValue' } },
+                    scheme: { value: 'schemeValue', question: 'Who?' },
+                },
+            ],
         };
-        const wrapper = enzyme.shallow(
-            <WizardInputs data={dummyData} />
-        );
+        const wrapper = enzyme.shallow(<WizardInputs data={dummyData} />);
         const instance = wrapper.instance();
-        const result = instance.renderInputs(dummyData.content,);
+        const result = instance.renderInputs(dummyData.content);
         expect(result[0]).not.toEqual(null);
     });
     it('should correctly generate captions for all restrictions', () => {
         const dummyData = {
             text: 'Category',
-            content: [{
-                test: {
-                    value: '',
-                    question: 'Why?',
-                    optional: true,
-                    maxLength: 40,
-                    lowercase: true,
-                    empty: true,
-                    problem: false
+            content: [
+                {
+                    test: {
+                        value: '',
+                        question: 'Why?',
+                        optional: true,
+                        maxLength: 40,
+                        lowercase: true,
+                        empty: true,
+                        problem: false,
+                    },
                 },
-            }],
+            ],
         };
-        const wrapper = enzyme.shallow(
-            <WizardInputs data={dummyData} />
-        );
+        const wrapper = enzyme.shallow(<WizardInputs data={dummyData} />);
         const instance = wrapper.instance();
         instance.renderInputElement('test', 0, dummyData.content[0].test);
-        expect(wrapper.find('FormField').props().caption).toEqual('Optional field; Field must be lowercase; Max length is 40 characters');
+        expect(wrapper.find('FormField').props().caption).toEqual(
+            'Optional field; Field must be lowercase; Max length is 40 characters'
+        );
     });
-    it('should handle select\'s onCLick', () => {
+    it("should handle select's onCLick", () => {
```

Comment:
Could we fix the indentation CLick instead of Click

Reply:
@balhar-jakub not seeing the indentation issue here?